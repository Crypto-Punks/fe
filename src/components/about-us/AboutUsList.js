import React from 'react';
import Developer from './AboutUsItem';
import { developerData } from '../../developerData';

const DeveloperList = () => {
  const developerElements = developerData.map((developer, i) => {
    return <Developer key={i} {...developer} />;
  });
  return (
    <ul>
      {developerElements}
    </ul>
  );
};


export default (DeveloperList);
