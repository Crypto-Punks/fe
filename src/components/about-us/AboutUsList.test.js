import React from 'react';
import { shallow } from 'enzyme';
import AboutUsList from './AboutUsList';

describe('AboutUsList component', () => {
  it('renders AboutUsList', () => {
    const wrapper = shallow(<AboutUsList />);
    expect(wrapper).toMatchSnapshot();
  });
});
