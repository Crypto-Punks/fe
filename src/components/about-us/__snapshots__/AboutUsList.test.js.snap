// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutUsList component renders AboutUsList 1`] = `
<ul>
  <Person
    blurb=""
    image="/src/images/developer-images/joe-klause.jpg"
    key="0"
    name="Joe Klause"
    spiritAnimal="Panther (or Squirrel...)"
    website="https://www.linkedin.com/in/joe-klause/"
  />
</ul>
`;
